src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1206,6 +1228,9 @@ namespace UnitarySystems {
             state.dataHVACGlobal->DXElecHeatingPower = 0.0;
             state.dataHVACGlobal->ElecHeatingCoilPower = 0.0;
         }
+        if (state.dataUnitarySystems->initUnitarySystemsErrorsFound) {
+            ShowFatalError(state, "Previous errors cause termination.");
+        }
```

Comment:
Added fatal at bottom of Init when initUnitarySystemsErrorsFound is true. Should be false unless set point is missing.

Reply:
I didn't like this if test at the bottom of init getting checked each iteration. Moved all logic to inside protected if block.